gdpr/anonymizers.py

```diff
@@ -170,8 +175,8 @@ def get_anonymized_value(self, value):
 
 
 class StaticValueAnonymizer(FieldAnonymizer):
-    """
-    Static value anonymizer replaces value with defined static value.
+    """Static value anonymizer replaces value with defined static value.
```

Comment:
stále nechápu proč jsi to udělal

Reply:
Protože jsi před 18 dny napsal, že:
"používáme už jiný docstring" a "google"... No tak jsem si to našel a změnil všechny komentáře v té knihovně...

Reply:
OK, upravil jsem je.